### ISSUE TITLE
problem with for(var key in obj) {...}

### ISSUE THREAD

**Issue by bultas** [speedskater/babel-plugin-rewire#18]
if i use your plugin with webpack configuration: `babel-loader?plugins=rewire`
and in my code is:

`for (var key in obj) {...}`

i get: 

*Module build failed: Error: /Users/.../findInObj.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?* error message

**Comment by vnovick** [comment]
Getting the same error. 


This is my example code:

```javascript
import { assert } from 'chai';
import React, { Component } from 'react';
import ComponentToTest from 'modules/welcomePanel/welcomePanel.component';

describe('Tests General Configuration', function(){
  it('should not be null', function(){
    ComponentToTest.__Rewire__('node',"hey I'm mock")
    let a = ['a','b','c'];
    for( let b of a){

    }
    let panel = new ComponentToTest
    panel.initPanel(document.body,"This is message")
    assert.isNotNull("Basic Test") 
})
})
ComponentToTest.__ResetDependency__('Card');
```

### package.json
```json
"dependencies": {
    "babel-core": "^5.7.0",
    "babel-loader": "^5.3.2",
    "babel-plugin-rewire": "^0.1.8",
    "babel-runtime": "^5.7.0",
```

### karma webpack configuration

```javascript
    webpack: {
          context: path.join(__dirname, 'frontend'),
          resolve: {
            modulesDirectories: ["node_modules", "scripts", "scripts/specs"],
            extensions: ["", ".js"],
          },
          entry: './scripts/specs/app.spec.js',
          devtool: "#inline-source-map",
          module: {
            loaders: [
              {
                test: /\.js$/,
                exclude: [
                 /node_modules/,
                 /translations\.js$/,
                ],
                loader: 'babel-loader',
                query: {
                 optional: 'runtime',
                 nonStandard: 'false',
                 plugins: 'rewire'
                },
              },
            ],
          }
        },
```

And this is the error I get:

```sh
ERROR in ./frontend/scripts/specs/app.spec.js
Module build failed: Error: /Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/frontend/scripts/specs/app.spec.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertAfter (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/path/modification.js:150:13)
    at NodePath.replaceWithMultiple (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/path/replacement.js:91:8)
    at NodePath.replaceWith (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/path/replacement.js:158:19)
    at NodePath.call (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/path/context.js:59:14)
    at NodePath.visit (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/path/context.js:89:8)
    at TraversalContext.visitSingle (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/context.js:131:12)
    at TraversalContext.visit (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/context.js:147:19)
    at Function.traverse.node (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/index.js:76:17)
    at NodePath.visit (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/path/context.js:106:26)
    at TraversalContext.visitMultiple (/Users/vlad/dev/VladDev/es6-webpack-babel-skeleton/node_modules/babel-core/lib/traversal/context.js:107:16)
webpack: bundle is now VALID.
```

Will appreciate the fix or if you can point where I can fix that will appreciate and fix.

**Comment by speedskater** [comment]
Thanks for the report and the example code. ) can reproduce your error and try to find a solution.

**Comment by speedskater** [comment]
I think I have fixed the issue in 0.1.11. Could you please verify if this resolves your issue?

**Comment by vnovick** [comment]
Yeah. It's fixed. Thanks. Fixed it too. just wanted to open a PR, but I've noticed that you fixed that.
